internal/provisioner/cluster.go

```diff
@@ -16,8 +17,13 @@ import (
 // clusterRootDir is the local directory that contains cluster configuration.
 const clusterRootDir = "clusters"
 
+// maxReadynessWaitSeconds controls how long we will wait for a kubernetes
+// cluster to become ready after it is created before we timeout and return
+// an error.
+const maxReadynessWaitSeconds = 600
```

Comment:
Is there a circumstance under which the client will want to control this parameter? Should we make `--wait` into an integer value with a default of `600`?

Reply:
That's a great idea!